Library/Services/Implementation/NfieldSurveyInterviewerAssignmentsService.cs

```diff
@@ -62,6 +63,18 @@ public Task UnassignAsync(string surveyId, string interviewerId)
             return Client.PutAsJsonAsync(uri, model).FlattenExceptions();
         }
 
+        /// <summary>
+        /// Implements <see cref="INfieldSurveyInterviewerAssignmentsService.PutAsync(string, SurveyInterviewerAssignmentModel)"/> 
```

Comment:
I imagine that the signature is wrong and should be (string, string, SurveyInterviewerAssignmentModel)
```